packages/node_modules/@ciscospark/react-container-message-composer/src/container.js

```diff
@@ -134,6 +134,9 @@ export class MessageComposer extends Component {
       onSubmit
     } = props;
 
+    if (activity.get('text').length === 0 && activity.get('files').size === 0) {
+      return;
+    }
```

Comment:
Does this check need to exist if the action has a check built in already?

Reply:
I went ahead and doubled up just to be safe in case we used the action elsewhere